__tests__/app.test.js

```diff
@@ -262,3 +262,34 @@ describe('/api/articles/:article_id', () => {
     });
 
 });
+
+describe('/api/articles/:articleid/comments', () => {
+    describe('POST comment', () => {
+
+        const insertObject =
+        {
+            username: 'icellusedkars',
+            body: 'I love coding...:)'
+        }
+
+        test.only('Trial ', () => {
```

Comment:
I would definitely tidy this up before being ready to go 👍

Reply:
Could you explain what exactly you mean, please? I think it's wiped in the last commit "ready to pull" :)

Reply:
I also cleared console.table()